packages/core/src/render3/instructions/styling.ts

```diff
@@ -127,17 +127,17 @@ export function ɵɵstyleMap(
 
 
 /**
- * Parse text as style and add values to ArrayMap.
+ * Parse text as style and add values to KeyValueArray.
  *
  * This code is pulled out to a separate function so that it can be tree shaken away if it is not
  * needed. It is only referenced from `ɵɵstyleMap`.
  *
- * @param arrayMap ArrayMap to add parsed values to.
+ * @param keyValueArray KeyValueArray to add parsed values to.
  * @param text text to parse.
  */
-export function styleStringParser(arrayMap: ArrayMap<any>, text: string): void {
+export function styleStringParser(keyValueArray: KeyValueArray<any>, text: string): void {
   for (let i = parseStyle(text); i >= 0; i = parseStyleNext(text, i)) {
-    styleArrayMapSet(arrayMap, getLastParsedKey(text), getLastParsedValue(text));
+    styleArrayMapSet(keyValueArray, getLastParsedKey(text), getLastParsedValue(text));
```

Comment:
I think this method needs to be `styleKeyValueArraySet`

```suggestion
    styleKeyValueArraySet(keyValueArray, getLastParsedKey(text), getLastParsedValue(text));
```